listenbrainz/webserver/static/js/src/recommendations/RecommendationCard.tsx

```diff
@@ -57,36 +228,87 @@ export default class RecommendationCard extends React.Component<
         <div className="col-xs-1 text-center">
           <div className="recommendation-controls">
             <>
-              <FontAwesomeIcon
-                icon={faEllipsisV as IconProp}
-                className="dropdown-toggle"
+              <button
+                className={`btn ${
+                  icon === faAngry
+                    ? "hate"
+                    : icon === faFrown
+                    ? "dislike"
+                    : icon === faMeh
+                    ? "bad"
+                    : icon === faSmileBeam
+                    ? "like"
+                    : icon === faGrinStars
+                    ? "love"
+                    : ""
+                }`}
```

Comment:
How about using ``className={`btn ${currentFeedback}`}`` here instead?
The only css class you would have to redefine is `bad`-> `bad_recommendation`.
With the added advantage of not requiring specifically these FA icons so they're easier to replace if we ever want to.